gamewin.cc

```diff
@@ -218,66 +206,69 @@ void Background_noise::handle_event(unsigned long curtime, uintptr udata) {
 					tracknum  = Audio::game_music(10);
 					laststate = DangerNear;
 				} else if (gwin->is_in_dungeon()) {
-					// Start the SFX music track then
-					tracknum  = Audio::game_music(52);
+					if (play_bg_tracks) {
+						tracknum = Audio::game_music(52);
+					}
 					laststate = Dungeon;
 				} else if (weather == 1) {    // Snowstorm
-					tracknum  = Audio::game_music(5);
+					if (play_bg_tracks) {
+						tracknum = Audio::game_music(5);
+					}
 					laststate = SnowStorm;
 				} else if (weather == 2) {    // Rainstorm
-					tracknum  = Audio::game_music(4);
+					if (play_bg_tracks) {
+						tracknum = Audio::game_music(4);
+					}
 					laststate = RainStorm;
-				} else if (bghour < 6 || bghour > 20) {
-					tracknum  = Audio::game_music(7);
+				} else if (nighttime) {
+					// Disabled nighttime sfx track as it is very distracting
+					/*if (play_bg_tracks)
+						tracknum  = Audio::game_music(7);*/
```

Comment:
I disagree with disabling this, at least like this. Maybe add a config option to disable it instead?